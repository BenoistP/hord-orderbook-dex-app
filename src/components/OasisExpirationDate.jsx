import React, { PureComponent } from 'react';
import { PropTypes } from 'prop-types';
// import ImmutablePropTypes from 'react-immutable-proptypes';

import './OasisExpirationDate.scss';


const propTypes = PropTypes && {
};
const defaultProps = {};


class OasisExpirationDate extends PureComponent {
  render() {
    return (
      <div className="OasisExpirationDate">
      </div>
    );
  }
}

OasisExpirationDate.displayName = 'OasisExpirationDate';
OasisExpirationDate.propTypes = propTypes;
OasisExpirationDate.defaultProps = defaultProps;
export default OasisExpirationDate;
