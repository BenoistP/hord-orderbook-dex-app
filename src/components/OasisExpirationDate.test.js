/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import { shallow } from 'enzyme';
import OasisExpirationDate from './OasisExpirationDate';

describe('(Component) OasisExpirationDate', () => {
  it('should render', () => {
    const props = {};
    const wrapper = shallow(
      <OasisExpirationDate {...props}/>,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
