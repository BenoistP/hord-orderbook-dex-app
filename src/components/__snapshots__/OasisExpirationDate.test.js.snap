// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) OasisExpirationDate should render 1`] = `
<div
  className="OasisExpirationDate"
>
  OasisExpirationDate
</div>
`;
